### PR TITLE
Update Quickstart.txt

### PR DESCRIPTION
**Pull request by WKFO**
Someone WILL most likely read Quicksave.txt and try to learn the game from there, so this is a miserable attempt to reduce the amount of "Why is X not working?" complaints a little. I never use default controls so if there is something wrong there, someone might want to take a look at that.

### CODE REVIEW

File: Quickstart.txt [hunk 1]
```diff
@@ -66,21 +65,26 @@ button.
 Use the cursor keys to move the camera in external and sidereal view. In
 sidereal view, use ',' (comma) and '.' (period) to roll the camera in place.
 
-To jump to another system, use the map (F2) and click on another system. Systems
-with ports are labelled in blue. You need enough Hydrogen to get there, and it
-must be close enough for your ship. F7 will start the hyperdrive.
+To jump to another system, use the map (F2) and click on another system. The
+map will provide you with info about the system, such as population, faction
```
Comment (nozmajner) [inline Quickstart.txt:69]: You also need to add it to your route. That caused confusion a couple of times if I remember correctly.

Comment (WKFO) [reply Quickstart.txt:69]: Ah, true. Good find.

Comment (WKFO) [reply Quickstart.txt:69]: I changed it, you may check if it is good.

Comment (nozmajner) [reply Quickstart.txt:69]: Sounds good.

**Comment by impaktor** [comment]
Good initiative. 

I note the linked wiki page, does not in turn link to any of these very useful pages:
https://pioneerwiki.com/wiki/Basic_flight
https://pioneerwiki.com/wiki/Flight_UI

I don't know if we want to minimize the number of wiki links in the quickstart? Or fi those three articles should reference each other?

**Pull request merged**